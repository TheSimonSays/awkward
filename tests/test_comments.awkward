# single line comment 0
print("done 0");
# single line comment 1
# single line comment 2
print("done 1 2");
# single line comment 3
print("done 3");


fn foo() {
    # foo function
    return "foo";
}

print(foo());
