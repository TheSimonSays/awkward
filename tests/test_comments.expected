done 0
done 1 2
done 3
foo
